qemu/tests/cfg/openflow_acl_test.cfg

```diff
@@ -14,7 +14,7 @@
             target_port = 80
             prepare_cmd_linux = echo "autotest page" > /var/www/html/autotest-test.html
             cleanup_cmd_linux = rm -rf /var/www/html/autotest-test.html
-            access_cmd_linux = lynx --dump http://ACCESS_TARGET/autotest-test.html
+            access_cmd_linux = wget http://ACCESS_TARGET/autotest-test.html --output-document=/dev/null
```

Comment:
@luckyh Thanks for review. I agree, already updated to curl. Please help to review again. Thanks.